taichi/transforms/scalarize.cpp

```diff
@@ -9,6 +9,23 @@
 
 namespace taichi::lang {
 
+template <typename T>
+static std::vector<Stmt *> get_scalarized_indices(T *stmt) {
```

Comment:
Why do we have to scalarize indices here? Have they been already scalarized at https://github.com/taichi-dev/taichi/blob/4f7f2f568d655e6fc507652f0d669c4aa077b75e/python/taichi/lang/impl.py#L166-L179